README.md

```diff
@@ -184,7 +184,7 @@ discontinued. For documentation on the older command options, refer to
 - `-h`, `--help`<br />
   Show commandline help, including a list of options, and sample use cases.
 
-- `-v`, `--version`<br />
+- `--version`<br />
```

Comment:
This is just a documentation update, the `-v` option was removed from the CLI argument parser elsewhere.